impl/src/main/java/org/eclipse/parsson/JsonMergePatchImpl.java

```diff
@@ -113,5 +113,38 @@ static JsonValue diff(JsonValue source, JsonValue target) {
         return builder.build();
     }
 
+    /**
+     * Compares this {@code JsonMergePatchImpl} with another object.
+     * @param obj the object to compare this {@code JsonMergePatchImpl} against
+     * @return true if the given object is a {@code JsonMergePatchImpl} with the same
+     * reference tokens as this one, false otherwise.
+     */
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj)
+            return true;
+        if (obj == null || obj.getClass() != JsonMergePatchImpl.class)
+            return false;
+        return patch.equals(((JsonMergePatchImpl)obj).patch);
```

Comment:
Would it make  sense to use `Objects.equals` (and `Objects.hashCode()` here?

Reply:
Done